package.json

```diff
@@ -9,7 +9,7 @@
     "install:packages": "pnpm --filter \"./packages/**\" install",
     "install:examples": "pnpm --filter \"./examples/**\" install",
     "lint": "eslint --cache .",
-    "lint:fix": "pnpm lint --fix",
+    "lint:fix": "pnpm lint --fix --ignore-path .gitignore --ignore-path .prettierignore",
```

Comment:
Should this be added to eslint config rather than inline?

Reply:
done